webapp/channels/src/components/advanced_text_editor/advanced_text_editor.tsx

```diff
@@ -311,11 +311,13 @@ const AdvancedTextEditor = ({
         setServerError,
         isInEditMode,
     );
+
+    const emojiPickerOffset = isInEditMode ? {right: 40} : undefined;
```

Comment:
Nitpick (and happy if we don't make any change):

I guess we always would put the same offset, so it would simplify the interface if we just pass a boolean to the hook saying "addOffset". That way, the consumer of `useEmojiPicker` doesn't have to think about how big the offset has to be.

But file is the only consumer of `useEmojiPicker`, so it is not really anything that important.